client/src/pages/Signup.js

```diff
@@ -0,0 +1,220 @@
+import React, { useState, useContext } from 'react'
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/core/styles';
+import * as imageURL from '../images/login-photo.png'
+import { Link } from 'react-router-dom'
+import Snackbar from '@material-ui/core/Snackbar'
+import SnackbarContent from '@material-ui/core/SnackbarContent'
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+      width: '25ch',
+      length: '400ch',
+      display: 'flex',
+      flexDirection: 'column',
+      marginBottom: '30px'
+    }
+  },
+}));
+
+const ContinueButton = withStyles({
```

Comment:
I noticed that these components are reused across multiple components. So, in this case we should be creating a folder for reusable components so we can reduce the repetitive code we have